db/migrate/20160324174132_create_deployments.rb

```diff
@@ -0,0 +1,23 @@
+class CreateDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string   :deployment_type
+      t.string   :deployment_version
+      t.boolean  :containerized, :default => true
+      t.string   :deployment_method
+      t.string   :deployed_provider_type
+      t.string   :deployed_provider_name
+      t.string   :ssh_private
+      t.string   :ssh_public
+      t.string   :ssh_user
+      t.string   :rhn_user
+      t.string   :rhn_pass
+      t.string   :rhn_sku
```

Comment:
So, the rhn_subscription role does not currently take a pool id, but we should allow for the specifying of a pool id to avoid issues when there are multiple SKUs that match the value passed in by the user.